server/sonar-server/src/test/resources/org/sonar/server/measure/custom/ws/MetricsActionTest/metrics.json

```diff
@@ -0,0 +1,34 @@
+{
+  "metrics": [
+    {
+      "key": "custom-key-1",
+      "name": "custom-name-1",
+      "type": {
+        "key": "INT",
+        "name": "Integer"
```

Comment:
Did you change your mind compared to what we discussed yesterday over the response format?